ejb3/src/main/java/org/jboss/as/ejb3/deployment/processors/ProhibitedEJBAnnotations.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2011, Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags. See the copyright.txt file in the
+ * distribution for a full listing of individual contributors.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+
+package org.jboss.as.ejb3.deployment.processors;
+
+import org.jboss.jandex.DotName;
+
+/**
+ * ProhibitedEJBAnnotations class. Class that stores the {@link DotName}s of EJB prohibited annotations.
+ *
+ * @author Eduardo Sant'Ana da Silva
+ */
+public enum ProhibitedEJBAnnotations {
+    /**
+     * javax.transaction.Transactional annotation. ejb-3_2 specification, section 7.1 : It is illegal to associate JTA
+     * transactional interceptors with Enterprise JavaBeans. The EJB Container should fail deployment of such applications.
+     */
+    TRANSACTIONAL(Constants.JAVAX_TRANSACTION, "Transactional"), TRANSACTIONAL_SCOPED(Constants.JAVAX_TRANSACTION,
```

Comment:
The spec only mentions transaction interceptors. TransactionScoped is a scope rather than an interceptor.